specification/supplychain/resource-manager/Microsoft.Hardware/preview/2023-07-12-preview/schemas.json

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
All new RPs should be defined using RPaaS. To create a new traditional RP, you need exception from RPaaS team, as explained here: [https://armwiki.azurewebsites.net/rp_onboarding/process/onboarding.html](https://armwiki.azurewebsites.net/rp_onboarding/process/onboarding.html). Unless yours is Ring 0 service. Please provide details if already provided exception.

Reply:
Hi @visingla-ms thank you for the feedback. We are trying to create the RP using RPaaS. I could not access the OpenAPI hub earlier so created this PR. I have created another PR via OpenAPIHub (https://github.com/Azure/azure-rest-api-specs-pr/pull/14845) - can this be reviewed by ARM instead?
Also what is prompting comment that this is not via RPaaS - is that because this PR is targeting main branch instead of RPSaaSDev or RPSaaSMaster branches?